asreview/io/csv_reader.py

```diff
@@ -20,6 +20,8 @@
 class CSVReader:
     """CVS file reader."""
 
+    name = "csv"
+    label = "CSV (UTF-8)"
```

Comment:
CSV reader supports also latin-1 (see below).

Reply:
Ah, I copied the label from the

Reply:
Yes, let's leave out.